homeassistant/components/mqtt/client.py

```diff
@@ -83,6 +83,8 @@
 _LOGGER = logging.getLogger(__name__)
 
 DISCOVERY_COOLDOWN = 2
+INITIAL_SUBSCRIBE_COOLDOWN = 1.0
```

Comment:
This value is never being used.  Did you mean to set the debounce time to this on connect/reconnect (or at the beginning of the discovery cooldown?)

Reply:
It is just to make sure that on slower systems all replayed items are finished first. Do not forget all services are starting up at that time.After that period we keep the debounce but with a shorter time.

Reply:
The constants can be mocked in the tests

Reply:
You are right, the debouncer isn't initialized with `INITIAL_SUBSCRIBE_COOLDOWN` as was requested by emontenemery.